CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Note
 
 - Removed `GenerateRequest` CRD.
+- Refactored `kyverno` chart, migration instructions are available in chart `README.md`.
```

Comment:
We need a sub-title to publish these to the release notes.  i.e., `##v1.10.0-rc1`.

Reply:
Do we really need this sub-title ? What if we release `v1.10.0-alpha.1` first ?

Reply:
Yes, see https://github.com/kyverno/kyverno/blob/fd0699c6d4411d701c3108147b2f6bf4ddffe9f4/Makefile#L682-L684.

Reply:
Fixed